libsoftwarecontainer/src/gateway/network/networkgateway.h

```diff
@@ -44,6 +44,12 @@ class NetworkGateway :
 public:
     static constexpr const char *ID = "network";
 
+    // Exit codes used when running inside container
+    static const constexpr int FAILURE = -1;
+    static const constexpr int SUCCESS = 0;
+    static const constexpr int NO_LINK = 1; // The desired link was not found
+    static const constexpr int BAD_LINKUPDOWN = 2; // Could not bring link up/down
```

Comment:
its kinda confusing maybe we can change the add two expression as BAD_LINKUP = 2 and BAD_LINKDOWN = 2